package.json

```diff
@@ -42,11 +42,13 @@
     "path": "~0.4.9",
     "q": "^1.0.1",
     "request": "2.34.x",
+    "request-promise-native": "^1.0.3",
```

Comment:
I added `supergaent` initially to add Promises support, and because we use it Northstar-JS and Phoenix-JS. Not sure if added another library is anything to flag, but would be curious about @sergii-tkachenko 's opinion